airbyte-integrations/connectors/destination-pubsub/src/main/resources/spec.json

```diff
@@ -0,0 +1,32 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/pubsub",
+  "supportsIncremental": true,
+  "supportsNormalization": false,
+  "supportsDBT": false,
+  "supported_destination_sync_modes": ["append"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Google PubSub Destination Spec",
+    "type": "object",
+    "required": ["project_id", "topic_name", "credentials_json"],
```

Comment:
one of the required field should be called `topic_id`

Reply:
Oops; forgot that part after rename; fixed!

Reply:
created a relevant issue: https://github.com/airbytehq/airbyte/issues/4352